android/src/main/java/com/auth0/react/A0Auth0Module.java

```diff
@@ -131,29 +129,28 @@ String generateCodeChallenge(@NonNull String codeVerifier) {
     }
 
     @Override
-    public void onHostResume() {
-        new Handler().postDelayed(new Runnable() {
-            @Override
-            public void run() {
-                Callback cb = A0Auth0Module.this.callback;
-                if (cb != null) {
-                    final WritableMap error = Arguments.createMap();
-                    error.putString("error", "a0.session.user_cancelled");
-                    error.putString("error_description", "User cancelled the Auth");
-                    cb.invoke(error);
-                    A0Auth0Module.this.callback = null;
-                }
+    public void onActivityResult(Activity activity, int requestCode, int resultCode, Intent data) {
+        Callback cb = A0Auth0Module.this.callback;
+
+        if (cb != null) {
```

Comment:
I'd flip this condition so we avoid indenting a big part of the code. Something like this is preferred:

if (cb == null){
return;
}

// the rest of the code here

Reply:
Changed in https://github.com/auth0/react-native-auth0/pull/350/commits/b5e827bce2820ffb8930272c044f1ccf12e5091e